code/game/turfs/open/_open.dm

```diff
@@ -107,6 +113,52 @@
 	. = ..()
 	update_visuals()
 
+//ATTACK GHOST IGNORING PARENT RETURN VALUE
+/turf/open/attack_ghost(mob/dead/observer/user)
+	if(destination_z)
+		var/turf/T = locate(destination_x, destination_y, destination_z)
+		user.forceMove(T)
```

Comment:
parent call?

Reply:
Hmm good point. Looks like this would break the ghost gas scan